### PR TITLE
Liel sa/download as excel

### PR DESCRIPTION
**Pull request by lielsa2023**
function for download as excel

### CODE REVIEW

**Comment by lielsa2023** [comment]
this is a function for dowload as excel

File: client-side/src/download as excl/downloadAsExcel.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import { saveAs } from 'file-saver';
+import * as XLSX from 'xlsx';
+
+export const downloadExcel = (data) => {  
+  const workbook = XLSX.utils.book_new();  
+  const worksheet = XLSX.utils.json_to_sheet(data);  
+  XLSX.utils.book_append_sheet(workbook, worksheet, 'Data');  
+  const excelBuffer = XLSX.write(workbook, {
+    bookType: 'xlsx',
```
Comment (e171271) [inline client-side/src/download as excl/downloadAsExcel.jsx:11]: where you are defined the name of the sheet and the name of the file?

File: client-side/src/download as excl/useDownLoadAsExcel.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+import React from 'react';
+import { downloadExcel } from './downloadAsExcel';
+
+const UseComponent = () => {
+  const data = [
+  { SiteName: 'Gmail', BrowsungTime: '150 h',Avg: '1 h' },
+  { SiteName: 'Netfree', BrowsungTime: '45 m',Avg: '0.04' },
+  { SiteName: 'Outlook', BrowsungTime: '15 m',Avg: '0.001' },
```
Comment (e171271) [inline client-side/src/download as excl/useDownLoadAsExcel.jsx:9]: the data shouldn't be hard-coded it should be taken from a state variable

**Review by e171271 — CHANGES_REQUESTED** [review]

File: client-side/src/download as excl/downloadAsExcel.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+
+import { saveAs } from 'file-saver';
+import * as XLSX from 'xlsx';
```
Comment (e171271) [inline client-side/src/download as excl/downloadAsExcel.jsx:4]: I don't see any change in package.json, you are using here a new libraries they should be added to package.json

**Review by e171271 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**